src/tools/spake2p/Cmd_GenVerifier.cpp

```diff
@@ -85,6 +86,19 @@ const char * const gCmdOptionHelp =
     "          * 12345678\n"
     "          * 87654321\n"
     "\n"
+    "   -f, --pin-code-file <file>\n"
+    "\n"
+    "       A file which contains all the PIN codes to generate verifiers.\n"
+    "       Each line in this file should be a valid PIN code. If the row count of this file is less than\n"
```

Comment:
Need to add that the PIN code is in the decimal number format.

Reply:
Added